src/js/themes/hpeWeb.js

```diff
@@ -0,0 +1,238 @@
+import { hpe } from './hpe';
```

Comment:
Think of another name instead of "hpe-web"

Reply:
Brainstorming:
- hpe-marketing
- hpe-com
- hpe-presentation
- hpe-not-product
- hpe-...

Reply:
Haha ... "not-product".

I sort of like "hpe-com", though that might be interpreted "compute". Wait, I just now get that it ties closely to "hpe.com", not bad. But, maybe not 1-1 aligned enough as app sites will be under "hpe.com" as well.

other ideas:

- hpe-pub
- hpe-site (based on https://medium.com/@essentialdesign/website-vs-web-app-whats-the-difference-e499b18b60b4)
- hpe-pres or hpe-prez
- hpe-typ

Reply:
haha "not-product" was definitely a brainstorm "no idea is a bad one" option to prompt thought/

agree re: "com" potentially being confused with compute or "compute ops manager"